plugins/repository-s3/src/main/java/org/opensearch/repositories/s3/S3ClientSettings.java

```diff
@@ -202,7 +201,7 @@ final class S3ClientSettings {
     static final Setting.AffixSetting<String> REGION = Setting.affixKeySetting(
         PREFIX,
         "region",
-        key -> new Setting<>(key, "", Function.identity(), Property.NodeScope)
+        key -> new Setting<>(key, "us-west-2", Function.identity(), Property.NodeScope)
```

Comment:
Would like to hear you opinion - in general, the region has to be  always specified, the `us-west-2` does not really reflect the common use case, I think we should make it empty and fail if not provided, what do you think?

Reply:
I think this is here because it was the default in AWS SDK v1.

Reply:
You mean when default was empty (before), the AWS SDK v1 used `us-west-2`? (just for own curiosity), +1 to replicate the default behaviour.

Reply:
@reta yes that is correct, in the v1 version, the default region was `us-west-2`, ref: https://github.com/aws/aws-sdk-java/blob/41355ccc7b4d9d05f545b75b77813ea6acd7e73b/aws-java-sdk-core/src/main/java/com/amazonaws/regions/Regions.java#L68

Currently, the `REGION` setting has a default value of an empty string, which case the v1 SDK will pick the default region.